src/com/namelessmc/java_api/Group.java

```diff
@@ -3,30 +3,31 @@
 import com.google.gson.JsonObject;
 
 public class Group implements Comparable<Group> {
-	
+
 	private final int id;
 	private final String name;
-	private int order;
+	private final int order;
 	private final boolean staff;
-	
+
 	Group(final JsonObject group) {
 		this.id = group.get("id").getAsInt();
 		this.name = group.get("name").getAsString();
-		this.staff = group.has("staff") ? group.get("staff").getAsBoolean() : false;
+		this.order = group.has("order") ? group.get("order").getAsInt() : -1;
```

Comment:
You can assume order is always present

Reply:
you totally forgot that, so I didn't know if it is always present or not